tools/gatk_intervallisttool.cwl

```diff
@@ -15,7 +15,7 @@ arguments:
     shellQuote: false
     valueFrom: >-
       set -eo pipefail
-      
```

Comment:
We talked in code review that this can probably be deleted. Maybe make a strategic week ticket to investigate if this (and other files) are actually needed and clean them up if they're not or add better descriptions to ones that are needed.

Reply:
Yeah I'll go through and delete what I can.